corehq/motech/repeaters/views/repeat_records.py

```diff
@@ -350,13 +357,26 @@ def cancel_repeat_record(request, domain):
 @requires_privilege_with_fallback(privileges.DATA_FORWARDING)
 def requeue_repeat_record(request, domain):
     if _get_flag(request) == 'select_cancelled':
-        _schedule_task_with_flag(request, domain, 'requeue')
+        try:
+            _schedule_task_with_flag(request, domain, 'requeue')
+        except BulkActionMissingParameters:
+            return _missing_parameters_response()
     else:
         _schedule_task_without_flag(request, domain, 'requeue')
 
     return HttpResponse('OK')
 
 
+def _missing_parameters_response():
+    return JsonResponse(
+        {
+            "failure_reason": _(
+                "Please filter to a specific repeater or payload before attempting a bulk action."
```

Comment:
Should this be translated?

Reply:
It seems like we typically do translate messages in JsonResponses, like this one is. So I'm going to say "yes".